keyboards/lets_split/keymaps/mekberg/keymap.c

```diff
@@ -0,0 +1,230 @@
+   |--------+--------+--------+--------+--------+--------|   |--------+--------+--------+--------+--------+--------|
+   |        |        |        |        |        |        |   |        |        |        |        |        |        |
+   `-----------------------------------------------------´   `-----------------------------------------------------'
+*/
+
+// Layers
+#define _DEFAULT 0
+#define _DEFAULT_MAC 1
+#define _DEFAULT_WIN 2
+#define _RAISE_MAC 3
+#define _RAISE_WIN 4
+#define _LOWER 5
+#define _SPECIAL 6
+
+#define TO_MAC TO(_DEFAULT)
+#define TO_WIN TO(_DEFAULT_WIN)
+
+#define RAISE_M MO(_RAISE_MAC)
+#define RAISE_W MO(_RAISE_WIN)
+#define LOWER MO(_LOWER)
+#define SPECIAL MO(_SPECIAL)
+
+// Modifiers
+#define SFTENT KC_SFTENT        // Enter => Shift when held
+#define CTLTAB LCTL_T(KC_TAB)   // Tab => Ctrl when held
+
+// ISO keycodes with Swedish layout names
+#define SE_PLUS KC_MINS         // +
+#define SE_QSTM S(SE_PLUS)      // Question mark
+#define SE_TCK KC_EQL           // Fronttick (´)
+#define SE_BTCK S(SE_TCK)       // Backtick (`)
+#define SE_UML KC_RBRC          // Umlaut (¨)
+#define SE_TAK S(SE_UML)        // Caret (^)
+#define SE_TILD RALT(SE_UML)    // Tilde (~)
+#define SE_QUT KC_NUHS          // Singlequote (')
+#define SE_DQUT S(KC_2)         // Doublequote (")
+#define SE_AST S(SE_QUT)        // Asterisk
+#define SE_AUML KC_QUOT         // Ä
+#define SE_ARNG KC_LBRC         // Å
+#define SE_OUML KC_SCLN         // Ö
+#define SE_LT KC_NUBS           // <
+#define SE_GT S(SE_LT)          // >
+#define SE_DASH KC_SLSH         // -
+#define SE_USCR S(SE_DASH)      // _
+#define SE_AT RALT(KC_2)        // @
+#define SE_EXCL S(KC_1)         // !
+#define SE_HASH S(KC_3)         // #
+#define SE_USD RALT(KC_4)       // $
+#define SE_PCNT S(KC_5)         // %
+#define SE_AMP S(KC_6)          // &
+#define SE_EQLS S(KC_0)         // =
+
+// Apple-specific layout keys
+#define AP_SLS S(KC_7)          // /
+#define AP_PIPE LALT(KC_7)      // |
+#define AP_BSLS A(AP_SLS)       // Backslash
+#define AP_LCBR A(AP_LPAR)      // {
+#define AP_RCBR A(AP_RPAR)      // }
+#define AP_LBR A(KC_8)          // [
+#define AP_RBR A(KC_9)          // ]
+#define AP_LPAR S(KC_8)         // (
+#define AP_RPAR S(KC_9)         // )
+
+// Windows-specific layout keys
+#define MS_SLS S(KC_7)          // /
+#define MS_PIPE RALT(SE_LT)     // |
+#define MS_BSLS RALT(SE_PLUS)   // Backslash
+#define MS_LCBR RALT(KC_7)      // {
+#define MS_RCBR RALT(KC_0)      // }
+#define MS_LBR RALT(KC_8)       // [
+#define MS_RBR RALT(KC_9)       // ]
+#define MS_LPAR S(KC_8)         // (
+#define MS_RPAR S(KC_9)         // )
+
+// My common shortcuts
+#define MY_PREV KC_MRWD         // Media: Previous
+#define MY_NEXT KC_MFFD         // Media: Next
+#define MY_PLAY KC_MPLY         // Media: Play/Pause
+#define MY_VOLU KC__VOLUP       // Media: Volume Up
+#define MY_VOLD KC__VOLDOWN     // Media: Volume Down
+#define MY_MUTE KC__MUTE        // Media: Mute
```

Comment:
Here too.

Reply:
Right, so the reason I use these codes is because the QMK configurator specifically says to use them for macOS, which is what I primarily use. Hence why I also define them as "MY_*" in the keymap. Anyone that uses another OS can then easily redefine them again and not have to replace them within the map itself.

Reply:
The "Multimedia Keys (macOS)" heading just means they only work on macOS. Any key or group of keys that does not specify an operating system after it can be assumed to work across the board.

Reply:
Fixed